frontend/src/lib/api.ts

```diff
@@ -886,6 +886,10 @@ class ApiRequest {
         return apiRequest
     }
 
+    public queryAsync(teamId?: TeamType['id']): ApiRequest {
+        return this.environmentsDetail(teamId).addPathComponent('query_async')
+    }
```

Comment:
Still I'm gonna say it's a bad idea to use "query_async" for something _on top of_ existing async querying. More appropriate would be `queryStream`/`query_stream`

Reply:
Or not "stream" anymore, since we're not streaming 🤔 More like `query_awaiting`… `query_v2`?

Reply:
Pushed `query_awaiting`, steppin on toes